lib/puppet/type/openldap_database.rb

```diff
@@ -83,6 +83,26 @@ def insync?(is)
           "{SSHA}" + Base64.encode64("#{Digest::SHA1.digest("#{should}#{salt}")}#{salt}").chomp == is
         when /^\{SHA\}.+/
           "{SHA}" + Digest::SHA1.hexdigest(should) == is
+        when /^\{SSHA(256|384|512)\}.+/
```

Comment:
Why is it `SSHA(256|384|512)` here but `SHA(256|384|512)` (SSHA vs SHA) on line 68?